sources/web/datalab/server.ts

```diff
@@ -156,8 +156,15 @@ function handleRequest(request: http.ServerRequest,
       (path.indexOf('/sessions') == 0)) {
 
     if (path.indexOf('/tree') == 0) {
-        loadedSettings[startup_path_setting] = path
-        settings_.updateUserSetting(userId, startup_path_setting, path, true);
+        const filePath = '/content' + path.substr('/tree'.length);
```

Comment:
nit: indentation is inconsistent in this file, it'd be nice to fix that while at it. Feel free to merge without.